File: DSP1_S3_JAVA/src/FiltroPrioridad.java
public class FiltroPrioridad implements Filtro{
    public void procesar(Formulario formulario){
        int edad = formulario.getEdad();

        if (edad >= 60)
            formulario.setPrioridad(Prioridad.ALTA);
        else if(edad >= 40 && edad < 60)
            formulario.setPrioridad(Prioridad.MEDIA);

        formulario.setEdad(edad);
    }
}

File: P1/P1S3/FiltroPrioridad.cpp
//
// Created by ines on 8/4/21.
//

#include "FiltroPrioridad.h"

FiltroPrioridad::FiltroPrioridad() {

}
void FiltroPrioridad::procesar(Formulario &formulario) {
    unsigned edad = formulario.getEdad();

    if(edad >= 60)
        formulario.setPrioridad(ALTA);
    else if(edad >= 40 && edad <60)
        formulario.setPrioridad(MEDIA);
}